pkg/visor/api.go

```diff
@@ -90,11 +91,16 @@ type Overview struct {
 	Transports      []*TransportSummary  `json:"transports"`
 	RoutesCount     int                  `json:"routes_count"`
 	LocalIP         string               `json:"local_ip"`
+	NATType         string               `json:"nat_type"`
+	PublicIP        string               `json:"public_ip"`
```

Comment:
Not sure if we need to provide a NAT type here. Maybe it would be sufficient to just return if it is available for `sudph` and `stcpr`.

Reply:
Okay

Reply:
About `public_ip`; i am currently sending the public IP and port together. Is the port required or should I send just the IP? @jdknives

Reply:
Also i haven't done any check regarding `sctpr` because it depends on only one side being on a public ip. So i think the NAT info should be send to the address resolver so it can be used to determin if the connection is possible before attempting it.